uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt5/client/grpc/GRPCControlServer.java

```diff
@@ -171,7 +178,8 @@ public void createMqttConnection(MqttConnectRequest request,
                     .keepalive(keepalive)
                     .cleanSession(request.getCleanSession())
                     .mqtt50(version == MqttProtoVersion.MQTT_PROTOCOL_V_50)
-                    .connectionTimeout(timeout);
+                    .connectionTimeout(timeout)
+                    .userProperties(userProperties);
```

Comment:
Do not call Builder.userProperties() when no user properties in the gRPC request